compiler-rt/lib/sanitizer_common/sanitizer_symbolizer_report.cpp

```diff
@@ -39,8 +39,12 @@ static bool FrameIsInternal(const SymbolizedStack *frame) {
                internal_strstr(file, "/include/c++/") ||
                internal_strstr(file, "/include/g++")))
     return true;
+  if (file && internal_strstr(file, "\\compiler-rt\\lib\\"))
+    return true;
   if (module && (internal_strstr(module, "libclang_rt.")))
     return true;
+  if (module && (internal_strstr(module, "clang_rt.")))
```

Comment:
is `libclang_rt.` redundant then?

Reply:
I mean not if you happened to build the library with that name....

It might show up for cygwin or mingw builds. I don't see a reason to do platform specific preprocessing here.

Reply:
new `if (module && (internal_strstr(module, "clang_rt.")))` will match existing `libclang`